notifications_api_common/models.py

```diff
@@ -63,6 +65,17 @@ def __str__(self):
             api_root=api_root
         )
 
+    def clean(self):
+        super().clean()
+        site_domain = get_domain()
+        if not site_domain.strip():
```

Comment:
I think is not necessary the `.strip()`
```suggestion
        site_domain = get_domain()
        if not site_domain:
```